docs/source/roadmap.rst

```diff
@@ -1,67 +1,52 @@
+* **At-scale refinements:** Ensure event handling reliability, and event storm resilience. Complete support for multi-node deployment of sensor containers and rules engines for resilience and throughput.
+* **Security hardening:** Complete security audit and address issues discovered so far.
 
-.. rubric:: Done in v1.1
+.. _content:
 
-* **FLOW:** Visual workflow representation and drag-and-drop workflow designer.
-* **RBAC:** Role based access control for packs, actions, triggers and rules.
-* **Pluggable auth backends** including PAM, Keystone, Enterprise LDAP.
-* **All-in-one installer**: production ready single-box reference deployment with graphical setup wizard.
-* **RHEL 6 and 7 support**
-* **Trace-tags**: ability to track a complete chain of triggers, rules, executions, related to a given triggering event.
-* **Native SSH:** replace Fabric; Fabric based SSH still available and can be enabled via config.
-* **WebUI major face-lift**
+Content & Integrations
+----------------------
 
+* **StackStorm Pack Exchange:** Make integration and automation packs discoverable, continuously tested, and community rated. Solve the problem of packs spread all over GitHub.
+* **Multi-node deployments:** Provide platform support for content deployment to multiple worker nodes, with better integration with git/GitHub. Simplify development and deployment of "automation as code" at scale.
+* **Multi target configurations for integration packs:** For a given integration pack, define and manage multiple targets. This should allow the user to choose which one of a set of configurations to use for a given action.
+* **First class Windows support:** switch to pywinrm for better license. Remote PowerShell via Powershell.REST.API. Windows-native ActionRunners. 
+* **Projects and Uber-flow:** Introduce projects to group and manage rules and workflows. Handle versions and dependencies. "Productize" flow-rule-flow-rule chain pattern, aka "uber-flow". Manage large number of automations across users and teams, on a single StackStorm deployment at enterprise scale.
+* **Action Output Structure Definition**: Enable optional definition of action payload, so that it can be inspected and used when passing data between actions in workflows.
+* **More integration packs:** push more content to the community to help work with most common and widely used tools. Tell us if there is tool you love and think we should integrate with, or better yet write a pack!.
 
-.. rubric:: Done in v0.11
+.. _rbac:
 
-* **ChatOps:** two-way chat integration beyond imagination.
-* **More integration packs**: Major integrations - Salt, Ansible, some significant others. `Check the full list <https://github.com/StackStorm/st2contrib/tree/master/packs>`_.
+RBAC
+----
 
-.. rubric:: Done in v0.9
+* **Filters**: Tag and property based filters, more refined and convenient access control
```

Comment:
here is an example for "no rubrics": filters on tags and properties are not in RBAC category but they touch it, as many others.